bouncer/sentry.py

```diff
@@ -0,0 +1,53 @@
+"""Report exceptions to Sentry/Raven."""
+from pyramid import tweens
+import raven
+
+from bouncer import __about__
+
+
+#: The Raven client object that's used to send reports to Sentry.
+#: This object is created once per app and re-used across requests.
+#:
+#: This should be accessed via request.raven rather than directly, because
+#: get_raven_client() below adds per-request context to the client.
+raven_client = raven.Client(release=__about__.__version__)
```

Comment:
It would make more sense to do this when the app is build in the `includeme` and shove it on the registry, as in `warehouse`...